src/server/models/Status.js

```diff
@@ -1,15 +1,20 @@
 import bookshelf from '../db';
 
-const TABLE_NAME = 'status';
+const TABLE_NAME = 'status_changes';
 
 class Status extends bookshelf.Model {
```

Comment:
@deepsadhi Okay, here's the thing, this table just stores a log of status changes that's why it's `status_changes`. There will be other tables `statuses` and `service_names` that instead will store statuses and service names we have. At the moment we're storing hard coded status & service names in the `status_changes` table only to avoid complexity at this moment. In the next release we need to address this and normalize the database.